### DISCUSSION TITLE
Experimentally determined permeabilities derived from in vitro assays calibration

### DISCUSSION THREAD

**Discussion by xutt0913** [Open-Systems-Pharmacology/Forum#1355] [Questions & Problems]
It is said that if I require an expert calibration of a defined set of experimentally determined permeabilities derived from in vitro assays, I can contact PK-Sim® support. Now I have the  Caco2 permeability value for apixaban, what other info is required for request of expert calibration?

**Comment by StephanSchaller** [comment]
Dear @xutt0913, this is a challenging topic across PBPK platforms. 
Please find an elaborate discussion on the topic here:
https://github.com/Open-Systems-Pharmacology/Forum/issues/23#issuecomment-308669758

In principle, you will need reference measures for probes that would fall into a similar chemical domain and for which you have a PBPK model to calibrate your parameterization. On top also account for efflux transporters.
